Gemfile

```diff
@@ -72,3 +72,7 @@ end
 # notifications
 gem "noticed", "~> 1.6.0"
 
+# Letter Opener Gem - View emails locally
+group :development do
```

Comment:
We should add this to the existing development group instead of having another block. See `web-console` inclusion.